zeroex/order_validator_test.go

```diff
@@ -240,3 +310,21 @@ func TestCalculateRemainingFillableTakerAmount(t *testing.T) {
 	remainingFillableTakerAssetAmount := calculateRemainingFillableTakerAmount(signedOrder, orderInfo, traderInfo)
 	assert.Equal(t, new(big.Int).Div(takerAssetAmount, big.NewInt(2)), remainingFillableTakerAssetAmount)
 }
+
+func modifyOrder(t *testing.T, order Order, field string, value interface{}) Order {
```

Comment:
Using a `string` to identify struct fields and an `interface{}` for values makes me nervous. I would rather have more lines of type-safe code than fewer lines of type-unsafe code. How do you feel about changing/removing this?

We could still use a type-safe `copyOrder` function so we don't have to create a new order from scratch every time: 

```go
// works because function arguments are passed as a copy and the Order type has no pointer fields.
func copyOrder(order Order) Order {
   return order
}
```